src/index.js

```diff
@@ -1,81 +1,40 @@
 /* istanbul ignore file */
 const generateDocFromSchema = require("./lib/generator");
-const path = require("path");
-const os = require("os");
-const GroupInfo = require("./lib/group-info");
-
-const DEFAULT_OPTIONS = {
-  schema: "./schema.graphl",
-  rootPath: "./docs",
-  baseURL: "schema",
-  linkRoot: "/",
-  homepage: path.join(__dirname, "../assets/", "generated.md"),
-  diffMethod: "SCHEMA-DIFF",
-  tmpDir: path.join(os.tmpdir(), "@edno/docusaurus2-graphql-doc-generator"),
-  loaders: {},
-  pretty: false,
-};
+const { buildConfig } = require("./config.js");
 
 module.exports = function pluginGraphQLDocGenerator(context, opts) {
-  // Merge defaults with user-defined options.
-  const config = { ...DEFAULT_OPTIONS, ...opts };
+  const isDefaultId = opts.id === "default";
+
+  const command = isDefaultId ? "graphql-to-doc" : `graphql-to-doc:${opts.id}`;
+  const description = isDefaultId
+    ? "Generate GraphQL Schema Documentation"
+    : `Generate GraphQL Schema Documentation for configuration with id ${opts.id}`;
```

Comment:
Cannot use `opts.schema` in the description because the schema might be overriden from the CLI `--schema` option later.

Reply:
Fine with it 👍